File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void swap(int []nums,int l,int r){
        int temp=nums[l];
        nums[l]=nums[r];
        nums[r]=temp;
    }
    public void moveZeroes(int[] nums) {
        int n=nums.length;
        int l=0;
        for(int r=0;r<n;r++){
            if(nums[r]!=0){
                swap(nums,l,r);
                l++;
            }
        }
    }
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def swap(self,nums:List[int],l:int,r=int)->None:
        nums[l],nums[r]=nums[r],nums[l]
    def moveZeroes(self, nums: List[int]) -> None:
        n=len(nums)
        l=0
        for r in range(n):
            if nums[r]!=0:
                self.swap(nums,l,r)
                l+=1
        
        